ParisBNF/abb/BNFabb57.xml

```diff
@@ -390,9 +402,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
           <history>
             <origin>
               <origDate notBefore="1400" notAfter="1499"/>
+              The manuscript was owned by the monastery <placeName ref="ETH1348Daga">Dāgā ʾƎsṭifānos</placeName>.
```

Comment:
INS ID

Reply:
What should I do here? Sorry, I do not understand.

Reply:
INS0329DE, not ETH1348Daga :) (ETH records are for "ethnic groups")

Reply:
!!!!! :)